install

```diff
@@ -42,7 +42,7 @@ download() {
 
   check_binary
 
-  local url=https://github.com/lotabout/skim/releases/download/v$version/${1}.tar.gz
+  local url=https://github.com/crodjer/skim/releases/download/v$version/${1}.tar.gz
```

Comment:
This should point to the `skim-rs` repo

Reply:
Oh yes, this should certainly not point to my fork. I will updated this patch.

Reply:
@khafatech , @LoricAndre I see quite a few changes in the way binaries are published, I can take up a refactor for this script if okay with you.
I don't use this for my ansible setup at present, but be okay to contribute this change.